shared/teams/new-team/wizard/new-team-info.tsx

```diff
@@ -0,0 +1,163 @@
+  const [description, setDescription] = React.useState(teamWizardState.description)
+  const [openTeam, setOpenTeam] = React.useState(
+    teamWizardState.name ? teamWizardState.open : teamWizardState.teamType === 'community'
+  )
+  const [showcase, setShowcase] = React.useState(
+    teamWizardState.name ? teamWizardState.showcase : teamWizardState.teamType !== 'other'
+  )
+  const [selectedRole, setSelectedRole] = React.useState<Types.TeamRoleType>(teamWizardState.openTeamJoinRole)
+  const [rolePickerIsOpen, setRolePickerIsOpen] = React.useState(false)
+
+  const continueDisabled = rolePickerIsOpen || teamNameTaken || name.length < 3
+
+  const onBack = () => dispatch(nav.safeNavigateUpPayload())
+  // TODO: consider using navigateUpTo instead to close the parent modal too.
+  const onClose = onBack
+  const onContinue = () =>
+    dispatch(
+      TeamsGen.createSetTeamWizardNameDescription({
+        description,
+        openTeam,
+        openTeamJoinRole: selectedRole,
+        showcase,
+        teamname: name,
+      })
+    )
+
+  return (
+    <Kb.Modal
+      onClose={onClose}
+      header={{
+        leftButton: <Kb.Icon type="iconfont-arrow-left" onClick={onBack} />,
+        title: <ModalTitle teamname="New team" title="Enter team info" />,
+      }}
+      footer={{
+        content: (
+          <Kb.Button label="Continue" onClick={onContinue} fullWidth={true} disabled={continueDisabled} />
+        ),
+      }}
+      allowOverflow={true}
+    >
+      <Kb.Box2 direction="vertical" fullWidth={true} style={styles.body} gap={'tiny'}>
+        <Kb.LabeledInput
+          placeholder="Team name"
+          value={name}
+          onChangeText={setName}
+          maxLength={16}
+          autoFocus={true}
+        />
+        {teamNameTaken ? (
+          <Kb.Text type="BodySmallError" style={styles.extraLineText}>
+            This team name is already taken
+          </Kb.Text>
+        ) : (
+          <Kb.Text type="BodySmall">
+            Choose wisely. Team names are unique and can't be changed in the future.
+          </Kb.Text>
+        )}
+        <Kb.LabeledInput
+          hoverPlaceholder="What is your team about?"
+          placeholder="Description"
+          value={description}
+          rowsMin={3}
+          rowsMax={3}
+          multiline={true}
+          onChangeText={setDescription}
+          maxLength={280}
+        />
+
+        <Kb.Checkbox
+          labelComponent={
+            <Kb.Box2 direction="vertical" alignItems="flex-start" style={styles.tallEnoughBox}>
+              <Kb.Text type="Body">Make it an open team</Kb.Text>
+              <Kb.Text type="BodySmall">Anyone can join without admin approval.</Kb.Text>
+              {openTeam && (
+                <Kb.Box2 direction="horizontal" gap="xtiny" alignSelf="flex-start" alignItems="center">
+                  <Kb.Text type="BodySmall">People will join as</Kb.Text>
+                  <FloatingRolePicker
```

Comment:
does this have a way of closing on click outside the boundaries? if so we should use it